docs/examples/layers-control/index.md

```diff
@@ -7,7 +7,7 @@ title: Layer Groups and Layers Control
 
 This tutorial will show you how to group several layers into one, and how to use the layers control to allow users to easily switch different layers on your map.
 
-{% include frame.html url="example.html" %}
+{% include frame.html width="100%" height="400" url="example.html" %}
```

Comment:
In HTML it's invalid to use % in `width` and `height` attributes, although some browsers seem to respect it. Just an FYI.

Reply:
The was left from a test that i was doing with setting value to attributes of the map. Thanks for letting me know. Removed them and the example is working as the rest.